visual-testing-app/src/application.tsx

```diff
@@ -1,72 +1,55 @@
-import { ComponentType, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+/// <reference types="vite/types/importMeta" />
```

Comment:
Is this line needed to use the `import.meta.globEager` function?

Reply:
Yes correct